content/how-to/account-management/account-management-faq/index.md

```diff
@@ -51,27 +50,22 @@ Per-billing and per-user usage are not offered at this time.
 
 {{< accordion title="Which users can create support tickets?" col="in" href="accordion22" >}}
 
-All users can create support tickets. However, only the account owner is
-updated on the ticket status.
+All users can create support tickets. However, only the account owner is updated on the ticket status.
```

Comment:
This is incorrect. Didn't we change this in a different PR, where I typed out that admins and device admins on dedicated accounts can create tickets. Device admins can only create tickets that have a device on them. Observers can't create tickets.... I explained all that somewhere. Where?

Reply:
Yes, changes are requested in PLATEXP-1084